draco/analysis/flagging.py

```diff
@@ -402,6 +406,205 @@ def process(self, timestream):
+            # Remove median at each frequency, if asked.
+            if self.remove_median:
+                for ff in range(nfreq):
+                    radiometer[ff, ii] -= np.median(radiometer[ff, ii][~origflag[ff]])
+            # Combine weights with static flag
+            start_flag = origflag | static_flag[:, None]
+            # Obtain MAD and TV masks
+            this_madmask, tvmask = self.mad_tv_mask(
+                radiometer[:, ii],
+                start_flag,
+                freq,
+                sigma=self.sigma,
+                tv_fraction=self.tv_fraction,
+            )
+            # combine MAD and TV masks
+            madmask.append(this_madmask | tvmask)
+
+            # Add TV channels to ST start flag.
+            start_flag = start_flag | tvmask
+            # Determine initial threshold
+            med = np.median(radiometer[:, ii][~start_flag])
+            mad = np.median(abs(radiometer[:, ii][~start_flag] - med))
+            rms = self.MADTORMS * mad
+            threshold1 = (
+                rms * self.start_threshold_sigma * self.max_m ** self.RMS_SCALING_DIFF
+            )
+
+            # SUMTHRESHOLD mask
+            stmask.append(
+                dracorfi.sumthreshold(
+                    radiometer[:, ii],
+                    self.max_m,
+                    start_flag=start_flag,
+                    threshold1=threshold1,
+                    correct_for_missing=True,
+                )
+            )
+
+        # Combine two polarization flags
+        madmask = np.logical_or(madmask[0], madmask[1])
+        stmask = np.logical_or(stmask[0], stmask[1])
+
+        if self.mask_type == "mad":
+            finalmask = madmask
+        elif self.mask_type == "sumthreshold":
+            finalmask = stmask
+        else:  # Combine ST and MAD masks
+            sidereal_day = ephemeris.SIDEREAL_S * 24 * 3600
+            timestamp = sensitivity.time
+            ntm = len(timestamp)
+            start_csd = ephemeris.unix_to_csd(timestamp[0])
+            start_ra = (start_csd - int(start_csd)) * (2.0 * np.pi)
+            ra_axis = (start_ra + (2.0 * np.pi) / ntm * np.arange(ntm)) % (2.0 * np.pi)
+            # Select Sun transit times
+            suntt = ephemeris.solar_transit(timestamp[0])  # Sun transit time
+            sunidx = np.argmin(abs(timestamp - suntt))
+            beam_window = (
+                2.0 * np.pi
+            ) / 90.0  # 4 deg in radians for the Sun. Each side.
+            # Mask the sun for the maximum aparent dec of ~24 deg
+            ra_window = beam_window / np.cos(np.deg2rad(24))
+            time_window = ra_window / (2 * np.pi) * sidereal_day
+            # Time spans where we apply the MAD filter.
+            madtimes = abs(timestamp - suntt) < time_window
+            # Select bright source transit times. Only CasA, CygA and TauA.
+            sources = [ephemeris.CasA, ephemeris.CygA, ephemeris.TauA]
```

Comment:
Ideally we could provide this list as a config parameter, but at the moment we don't have any similar functionality in `draco`. For the moment you might just need to replace this with a list of RA and DECs.